axum-extra/src/typed_header.rs

```diff
@@ -156,6 +156,9 @@ impl TypedHeaderRejectionReason {
 
 impl IntoResponse for TypedHeaderRejection {
     fn into_response(self) -> Response {
+        let status = StatusCode::BAD_REQUEST;
+        let body = self.to_string();
+        axum_core::__log_rejection!(rejection_type = Self, body_text = body, status = status);
         (http::StatusCode::BAD_REQUEST, self.to_string()).into_response()
```

Comment:
```suggestion
        (status, body).into_response()
```

Surely `__log_rejection!` does not consume either of these bindings, right?

Reply:
Also, looks like you need to undo my change, oops 😅

If you put the macro args on separate lines, does rustfmt undo that?

Reply:
Right, this should use the variables. The whole point was to not define them twice, this is just my mistake.

Reply:
Feel free to commit the change to finish this off, I don't see the button to do so in the mobile app and won't be at a computer until tomorrow.